.circleci/config.yml

```diff
@@ -0,0 +1,78 @@
+version: 2.1
+
+jobs:
+  check:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            # when lock file changes, use increasingly general patterns to restore cache
+            - node-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
+            - node-v1-{{ .Branch }}-
+            - node-v1-
+      - run:
+          name: "Install deps"
+          command: npm ci
+      - save_cache:
+          paths:
+            - ./node_modules
+          key: node-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
+      - run:
+          name: "Commit linting"
+          command: |
+            git config --global user.email "you@example.com"
+            git config --global user.name "Your Name"
+            npx tasegir commitlint --circleci
+      # - run:
+      #     name: "Dependency check"
+      #     command: npx tasegir dep-check
+      - run:
+          name: "Code linting"
+          command: npx tasegir lint
+
+  node-tests:
+    parameters:
+      version:
+        type: string
+    docker:
+      - image: circleci/node:<< parameters.version >>
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            # when lock file changes, use increasingly general patterns to restore cache
+            - node-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
+            - node-v1-{{ .Branch }}-
+            - node-v1-
+      - run:
+          name: "Node tests"
+          command: npx truffle test
+#      - sonarcloud/scan
+
+#orbs:
+#  sonarcloud: sonarsource/sonarcloud@1.0.1
+
+workflows:
+  test:
+    jobs:
+      - check
+      - node-tests:
+          version: '10'
+          name: node-10
+#          context: SonarCloud
+          requires:
+            - check
+      - node-tests:
+          version: '12'
```

Comment:
We don't have to test multiple Node versions, it will only delay the CI run. I guess v12 should be sufficient.

Reply:
I guess you are right, my though process was that the contracts will be included in frontend but actually it is just the compiled JSON's.